src/notifiers/script.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from models import Config, Item
+from models.errors import ScriptConfigurationError, MaskConfigurationError
+from notifiers import Notifier
+
+log = logging.getLogger('tgtg')
+
+
+class Script(Notifier):
+    """Notifier for the script output"""
+
+    def __init__(self, config: Config):
+        self.enabled = config.script.get("enabled", False)
+        self.command = config.script.get("command")
+        self.timeout = config.script.get("timeout", 60)
+        self.cron = config.script.get("cron")
+
+        if self.enabled:
+            try:
+                Item.check_mask(self.command)
+            except MaskConfigurationError as exc:
+                raise ScriptConfigurationError(exc.message) from exc
+
+
+    def _send(self, item: Item) -> None:
+        import subprocess
+        commandline = self.command
+        commandline = commandline.replace('${{','~${{')
+        commandline = commandline.replace('}}','}}~')
```

Comment:
What is the reason for this?

Reply:
The problem that I faced while testing is that if the unmasked command line item contains spaces (e.g. display_name), the separation of the command line arguments goes wrong. That's why I added a tilde sign before and after the accolades before unmasking.